src/shared/resourcefetcher/httpResourceFetcher.ts

```diff
@@ -175,3 +175,14 @@ export async function getPropertyFromJsonUrl(
         }
     }
 }
+
+/**
+ * Gets contents from a remote resource
+ * @param url url of the resource
+ * @param fetcher optional HTTP resource fetcher to use
+ * @returns contents from the url if available or undefined
+ */
+export async function getFromUrl(url: string, fetcher?: HttpResourceFetcher): Promise<string | undefined> {
+    const resourceFetcher = fetcher ?? new HttpResourceFetcher(url, { showUrl: true })
+    return resourceFetcher.get()
```

Comment:
Looks like the `fetcher` param doesn't make sense, because it doesn't use `url`.